webapp/src/serial.tsx

```diff
@@ -475,7 +475,7 @@ export class Editor extends srceditor.Editor {
                     </div>
                 </div>
                 {this.charts?.length == 0 && <div id="serialPlaceholder" className="ui segment">
-                    <div className="ui orange bottom left attached no-select label seriallabel">0</div>
+                    <div className="ui bottom left attached no-select label seriallabel">{lf("Values will be logged when the device or simulator receives data")}</div>
```

Comment:
```suggestion
                    <div className="ui bottom left attached no-select label seriallabel">{lf("Values will be logged when the {0} sends data", this.isSim ? lf("simulator") : lf("device"))}</div>
```

^^ would that make sense? I think only showing simulator / device makes sense, sends vs receives I'm not certain on but I think of it as the simulator sending / creating the data personally?

Reply:
sounds good to me!